rj_gameplay/rj_gameplay/play/penalty_defense.py

```diff
@@ -18,26 +17,27 @@ class State(Enum):
 
 
 class PenaltyDefense(stp.play.Play):
-    """
-    Play consists of:
-                    -5 robots lined up on the side
-                    -1 goalie
-    The goalie will defend the opponents penalty kick.
-    """
-
     def __init__(self):
         super().__init__()
+
         self._state = State.INIT
 
     def tick(
         self,
         world_state: stp.rc.WorldState,
     ) -> List[RobotIntent]:
+
         if self._state == State.INIT:
             self.prioritized_tactics.append(goalie_tactic.GoalieTactic(world_state, 0))
-            self.prioritized_tactics.append(line_tactic.LineTactic(world_state, 5))
+            num_liners = len(world_state.our_visible_robots) - 1
+            start_pt = np.array([-3.0, 0.5])
```

Comment:
We need to change the start point and end point for the line tactic, but that is in a different PR.

Reply:
put it on the board!